go.mod

```diff
@@ -63,6 +63,7 @@ require (
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
 	github.com/opencontainers/image-spec v1.0.1 // indirect
 	github.com/optiopay/kafka v0.0.0-00010101000000-000000000000
+	github.com/pbnjay/memory v0.0.0-20201129165224-b12e5d931931
```

Comment:
I think we can just create our own pkg instead of pulling this dependency for a few LOCs: https://github.com/pbnjay/memory/blob/master/memory_linux.go.

Reply:
Even though the code that the package implements is quite small, the reason I went with the package is because it's cross-platform and that allows developers to run the Maglev unit tests. We have a few members who use OSX and as the team expands, maybe others come in with FreeBSD or something. WDYT?

Reply:
We already should vendor a repo in for it, see `mem.VirtualMemory()` in `calculateBPFMapSizes()`, I presume this should work as well here. (In general, my preference would be to vendor less external repos even if a bit more work for us.)

Reply:
Ah good catch, thanks. I wasn't aware we already had a library for this. Dropping the vendor commit